### ISSUE TITLE
Cannot navigate through folders using keyboard arrows

### ISSUE THREAD

**Issue by akanailov** [microsoft/AzureStorageExplorer#6318]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.26.1

### Architecture

x64

### Storage Explorer Build Number

20221018.5

### Platform

Windows

### OS Version

Windows 11 Enterprise

### Bug Description

I cannot scroll through the folders using keyboard arrows. The arrows navigate through the folders, but the screen doesn't follow the current folder. I have to scroll using my mouse to find the selected folder.

### Steps to Reproduce

1. Launch storage explorer
2. Open a blob container with many folders (so that they don't fit on the screen)
3. Try to scroll down using keyboard arrows.

### Actual Experience

<img width="424" alt="image" src="https://user-images.githubusercontent.com/119638887/205192319-dc711ae2-cea8-4744-bd2a-e3348e3710a8.png">
You can see on the picture that the screen doesn't follow the selected folder. If I scroll further down, I won't see which folder is selected.

### Expected Experience

I expect the screen to follow the selected folder, just like in the regular Windows File Explorer.

**Comment by JasonYeMSFT** [comment]
I made the change for 1.28.0 release to make the scroll location update when you navigate using the up/down arrow keys.
